### PR TITLE
MM-56513 Prevent deactivated users to be added as channel members

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
The API allows adding deactivated users as channel members, this PR fixes that

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-56513

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by lindy65 — APPROVED** [review]
Thanks @enahum - this is fixed 👍

File: server/i18n/en.json [hunk 1]
```diff
@@ -4850,6 +4850,10 @@
     "id": "app.bot.permenent_delete.bad_id",
     "translation": "Unable to delete the bot."
   },
+  {
+    "id": "app.channel.add_member.deleted_user.app_error",
+    "translation": "Unable to add a deactivated user as a member of the channel."
```
Comment (sbishel) [inline server/i18n/en.json:4855]: Should this error be less descriptive? Information disclosure... I know its a valid user id of a deactivated user.

**Review by sbishel — APPROVED** [review]
Approving just unsure about error details.

File: server/i18n/en.json (hunk 1 above)
Comment (enahum) [reply server/i18n/en.json:4855]: Nice! Well you tell me, I'm not really sure what else to use instead.

**Pull request merged**